examples/image_detection.py

```diff
@@ -6,15 +6,35 @@
 
 from trame_annotations.widgets.annotations import ImageDetection
 
+ANNOTATIONS = [
+    {
+        "id": 1,
+        "category_id": 0,
+        "label": "if matching category, should not be shown",
+        "bbox": [60, 50, 100, 100],  # xmin, ymin, width, height  <-- COCO format
+    },
+    {
+        "id": 99,
+        "category_id": 1,
+        "label": "fallback label",
+        "bbox": [140, 100, 100, 100],
+    },
+]
+
+CATEGORIES = [{"id": 1, "name": "my category"}]
+
 
 @TrameApp()
 class ImageDetectionExample:
     def __init__(self, server=None):
         self.server = get_server(server, client_type="vue3")
-        self._build_ui()
 
+        self.server.state.annotations = ANNOTATIONS
+        self.server.state.categories = CATEGORIES
         self.server.state.selected_id = ""
```

Comment:
you can remove those 2 lines since you did that assignment in the template.

Reply:
✅